### PR TITLE
Fix an "Ambiguous URI empty segment" error

### PR DESCRIPTION
**Pull request by terales**
* Fixed an "Ambiguous URI empty segment" error that wasn't clear for users how to workaround.
  * The problem was that direct calls from `AppInvocable` weren't stripping the trailing slash like `OkapiClient` did.
  * I've fixed it by extracting file upload requests into a common method within `OkapiClient` instead of two direct calls from `AppInvocable`.
* Added a first integration test.
* Cleared up trailing whitespaces.

### CODE REVIEW

**Comment by terales** [comment]
@vitalii-bezuhlyi @mathijs-bb can you review, please?
This is my first PR to the BlackBird's app, so comment on anything unusual, please.

File: Apps.Okapi/Api/OkapiClient.cs [hunk 1]
```diff
@@ -16,25 +16,42 @@ public async Task<T> ExecuteWithXml<T>(string endpoint, Method method, object? b
...
+        var baseUrl = creds.Get(CredsNames.Url).Value
+            .TrimEnd('/');
+
         var request = new OkapiRequest(new()
         {
             Url = baseUrl + endpoint,
             Method = method
         });
 
+        request.AddFile(fileParam.Name, () => fileParam.GetFile(), fileParam.FileName, fileParam.ContentType);
```
Comment (terales) [inline Apps.Okapi/Api/OkapiClient.cs:50]: For unknown reason, document says it's possible to pass a `FileParameter` to `request.AddFile`, but this overload is not actually implemented 🤷‍♂️

File: Tests.Okapi/Base/TestBase.cs
Comment (terales) [inline Tests.Okapi/Base/TestBase.cs:1]: Copy/paste from OpenAI app

File: Tests.Okapi/Base/FileManagementClient.cs
Comment (terales) [inline Tests.Okapi/Base/FileManagementClient.cs:1]: Copy/paste from OpenAI app

File: Apps.Okapi/Api/OkapiClient.cs (hunk 1 above)
Comment (terales) [reply Apps.Okapi/Api/OkapiClient.cs:50]: I've covered it with test and just doubled checked in a fly, it succeeded:
https://us-1.blackbird.io/flights/681a758c8b773a2feeca729f

Do you have any specific edge case in mind where it could fail?

**Review by vitalii-bezuhlyi — APPROVED** [review]
